dev_tools/scripts/deploy-cega.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -e
+
+ls -la LocalEGA-helm/ega-charts/cega/config
+
+cat LocalEGA-helm/ega-charts/cega/config/trace.yml
```

Comment:
If you want this for debugging reasons put them in an if block so that they are triggered only if debugging is selected.

```
if [ $1 -eq "debug" ]; then
```

Reply:
This was there already. Not adding anything new

Reply:
Doesn't matter if they where there or not.
They are clearly for debugging so either wrap them in a if clause or remove them